src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs

```diff
@@ -40,6 +40,56 @@ public async Task InsertMutation(string dbQuery)
             SqlTestHelper.PerformTestEqualJsonStrings(expected, actual.ToString());
         }
 
+        /// <summary>
+        /// <code>Do: </code> Inserts new row and return all its columns
+        /// <code>Check: </code> A row is inserted in the table that has rows with default values as built_in methods.
+        /// it should insert it correctly with default values correctly handled by database.
+        /// current_date, current_timestamp, random_number, next_day have default value as built_in methods GETDATE(), NOW(), RAND(), DATEADD(), resp.
+        /// default_string_with_parenthesis has default value "()", default_function_string_with_parenthesis has default value "NOW()".
+        /// default_integer has default value 100, default_date_string has default value "1999-01-08 10:23:54".
```

Comment:
need to mention how adding excluded fields to runtime config permissions affects this test and what is expected. Also update pr description describing how the tests address / cover the case outlined in the customer raised issue.

Reply:
i will update the summary highlighting that the test captures the scenario of using fields that have default built_in methods irrespective of it being in the list of excluded fields

Reply:
The issue was discovered  due to excluding fields that used functions as default values. So need to explain how fixing this issue also fixes the experience/bug encountered when referencing fields that define default values with functions.